lib/msal-native-auth/src/error/NativeAuthApiError.ts

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Generally these classes are missing unit tests. We need to write unit tests along with the class definitions as that will shape the form of the code.

Reply:
We have discussed offline.